### ISSUE TITLE
Text review in new Wireless configurator of Thymio Suite

### ISSUE THREAD

**Issue by cbarraud** [Mobsya/aseba#609]
In the new configuration tools, I propose to do some text modifications:

### In the tools of Thymio Suite
![image](https://user-images.githubusercontent.com/4930020/63837872-94239b80-c97c-11e9-93e9-2727f45d1f2c.png)

- [ ] "Pair a Wireless Thymio to a Wireless dongle" 
- [ ] "Pair a group of Wireless Thymio robots independently"

### On the pop-up window before the configurator
![image](https://user-images.githubusercontent.com/4930020/63843290-70fde980-c986-11e9-9dc7-67613ae77d43.png)

- [x] "Open the Wireless Thymio network configurator"
- [x] "All Wireless Thymio robots associated to the Wireless dongle plugged on this computer will be disconnected until the Wireless Thymio network configuration window is closed."

### In the Pair a Thymio Wireless to a dongle interface
![image](https://user-images.githubusercontent.com/4930020/63843483-bcb09300-c986-11e9-9be5-11c912311ec9.png)

- [x] "Wireless Thymio network configurator"
- [x] "Plug the Wireless Thymio robot you want to pair using the USB cable"
- [x] "Plug the Wireless dongle"
- [x] "Please plug a Wireless Thymio robot with a USB cable to this computer"

### In the Advanced mode interface
![image](https://user-images.githubusercontent.com/4930020/63843909-77409580-c987-11e9-918f-8958adb5a4ca.png)
- [x] "Wireless Thymio network configurator"
- [x] "To pair multiple Wireless Thymio robots to one Wireless dongle:"
- [x] "03. Click on Pair !"

### In the pair a case of thymios interface
![image](https://user-images.githubusercontent.com/4930020/63844088-d6060f00-c987-11e9-9f05-732dca7f80b0.png)

- [x] "Wireless Thymio network configurator"
- [x] "Plug the Wireless Thymio robot you want to pair using the USB cable"
- [x] "Plug the Wireless dongle"
- [x] "Tip: Put a sticker dot of the same color on both the Wireless Thymio robot and the Wireless dongle to identify them!
(open question, should we advise the color dot sticker or the serial number of the robot? could depends also of next move in production)

**Comment by mbonani** [comment]
for me all comments are going to our standart. 
just for the case I am not sure if it is clear what means "Pair a group of Wireless Thymio robots independently""

**Comment by mbonani** [comment]
>  (open question, should we advise the color dot sticker or the serial number of the robot? could depends also of next move in production)
to see with @riedo but basic idea was that we add stickers in the cases

**Comment by cbarraud** [comment]
> just for the case I am not sure if it is clear what means "Pair a group of Wireless Thymio robots independently""

I admit that it's not very good :) a bit wordy too. 

"Pair several robots independently" ?
"Pair several Wireless Thymio robots each to one Wireless dongle" ?
"Pair a Wireless Thymio suitcase" ?

maybe the last one is good enough?

**Comment by mbonani** [comment]
"Pair a Wireless Thymio case" we have to be simple
to be check but it seems "suitcase" is not the correct term in English, in french for sure it will be "Appairer une valise Wireless Thymio"

**Comment by mbonani** [comment]
"Pair a case of Wireless Thymio" like in the PR is also good for me
